src/Serialize.jl

```diff
@@ -1,19 +1,20 @@
 """
 `savejets(filename, jets; format="px py pz E")`
 
-Saves the given `jets` into a file with the given `filename`. Each line contains information about a single jet and is formatted according to the `format` string which defaults to `"E px py pz"` but can also contain other values in any order: `"pt"` or `"kt"` for transverse momentum, `"phi"` for azimuth, `"eta"` for pseudorapidity, `"m"` for mass. It is strongly NOT recommended to put something other than values and (possibly custom) separators in the `format` string.
+Saves the given `jets` into a file with the given `filename`. Each line contains information about a single jet and is formatted according to the `format` string which defaults to `"px py pz E"` but can also contain other values in any order: `"pt2"` or `"kt"` for transverse momentum, `"phi"` for azimuth, `"eta"` or `"rapidity"` for pseudorapidity, `"m"` for mass. It is strongly NOT recommended to put something other than values and (possibly custom) separators in the `format` string.
 """
 function savejets(filename, jets; format="px py pz E")
     symbols = Dict(
         "E" => JetReconstruction.energy,
         "px" => JetReconstruction.px,
-        "pt" => JetReconstruction.pt,
-        "kt" => JetReconstruction.pt,
+        "pt2" => JetReconstruction.pt2,
+        "kt" => JetReconstruction.pt2,
```

Comment:
should "kt" => pt?

Reply:
pt() is no more required to be defined and there's now only pt2. I understand the confusion though

Reply:
if it's absurd from the standpoint of physics then we could remove kt from the map entirely

Reply:
I thought "kt" is transverse momentum, thus should equal to pt not pt2 (which is sqaure of that?)

Reply:
well, as I've said I forgot how this works because I haven't been doing this physics in a while :)
I think addressing both of your comments, I could just leave it as it was, since it won't really crash unless someone tries to use "kt" in formatting without defining the kt() method for their type

Reply:
nevermind, it actually will crash. I'll just stick to the set of required methods then: pt2(), phi(), rapidity(), px(), py(), pz(), energy()